tutorials/napps/publishing_your_napp.rst

```diff
@@ -179,49 +172,52 @@ Uploading the NApp
 ******************
 
 Your NApp is now ready to be uploaded. 
-Before that, if you do not want to send some files to the server, add them to .gitignore.
-To upload your Napp, use the following command:
+Before that, we need to create a repository with NApp name on github. 
+After creating repository follow the below commands to initialize and push your NApp to github.
 
 .. code-block:: console
 
   $ cd ~/tutorials/<username>/my_first_napp
-  $ kytos napps upload
-  Enter the username: <username>
-  Enter the password for <username>: <password>
-  SUCCESS: NApp <username>/my_first_napp uploaded.
+  $ git init
+  $ git remote add origin https://github.com/<yourGithubAccountName>/<repoName>.git
```

Comment:
I would suggest `yourGithubAccountName` -> `username` and  `repoName` -> `name`, to use the same naming consistency throughout the docs